docs/onboarding/azure-devops-pipelines.md

```diff
@@ -421,22 +467,24 @@ In order to configure audit stream for Azure Monitor, identify the following inf
     is optional since the deployment will be based on the pipeline you configure anyway. So if you configured the **platform-connectivity-hub-azfw-policy-ci** and the **platform-connectivity-hub-azfw-ci** pipelines,
     only the _Azure Firewall_ option will be deployed, not the _NVA_ option.
 
-    * Update **var-hubnetwork-managementGroupId** with the networking management group.  This is based on the prefix defined in `var-topLevelManagementGroupName`.  For example, if `var-topLevelManagementGroupName` is set to `contoso`, then `var-hubnetwork-managementGroupId` will be `contosoPlatformConnectivity`.
-    
+    * Update **var-hubnetwork-managementGroupId** with the networking management group:
+        * For **CanadaPubSecALZ v0.9.0 or later**, this is based on the `id` attribute of the first child element defined in your `var-managementgroup-hierarchy` configuration value. In JSON notation, this is equivalent to: `.children[0].id`.
```

Comment:
This wouldn't be the case when we have the hierarchy.  Let's rephrase it to say that with v0.9.0 or later, this will be the management id that represents the Networking Management Group in the defined hierarchy.

Reply:
Good catch; I was a little too quick with copy/paste from higher up in the document. Fixed.